src/GoogleAdWordsEventForwarder.js

```diff
@@ -53,6 +87,24 @@
 
                 try {
                     if (window.gtag && forwarderSettings.enableGtag == 'True') {
+                        var eventConsentState = getEventConsentState(event);
+
+                        if (eventConsentState) {
+                            var updatedConsentPayload = generateConsentStatePayloadFromMappings(
+                                eventConsentState,
+                                consentMappings,
+                                consentPayloadDefaults,
+                            );
+    
+                            var eventPayloadAsString =
+                                JSON.stringify(updatedConsentPayload);
+    
+                            if (eventPayloadAsString !== consentPayloadAsString) {
+                                sendGtagConsentUpdate(updatedConsentPayload);
+                                consentPayloadAsString = eventPayloadAsString;
```

Comment:
Quick variable rename: It's not the event payload, it's the event's consent.  
```suggestion
                            var eventConsentAsString =
                                JSON.stringify(updatedConsentPayload);
    
                            if (eventConsentAsString !== consentPayloadAsString) {
                                sendGtagConsentUpdate(updatedConsentPayload);
                                consentPayloadAsString = eventConsentAsString;
```